packages/react/src/NoteDisplay/NoteDisplay.tsx

```diff
@@ -0,0 +1,41 @@
+import { Blockquote, createStyles, Stack } from '@mantine/core';
+import { Annotation } from '@medplum/fhirtypes';
+import React from 'react';
+
+const useStyles = createStyles((theme) => ({
+  noteBody: { fontSize: theme.fontSizes.sm },
+  noteCite: { fontSize: theme.fontSizes.xs, marginBlockStart: 3 },
+  noteRoot: { padding: 5 },
+}));
+
+export interface NoteDisplayProps {
+  value?: Annotation | Annotation[];
```

Comment:
I believe that every use of `<NoteDisplay>` uses `Annotation[]` array, never single `Annotation`.  Any objections to simplifying and just taking `Annotation[]`?

Reply:
I actually originally had it like this, but then was worried that the name `Note` (singular) would imply that non-arrays would be ok. 

But `NotesDisplay` seemed like a weird name. 

I'll change it to avoid the perf hit of the array check